sherpa/astro/tests/test_astro_data2.py

```diff
@@ -4205,3 +4202,162 @@ def test_pha_ungroup_background_remove(caplog):
     assert not bkg.grouped
 
     assert len(caplog.record_tuples) == 0
+
+
+def test_pha_check_background_ids_basic():
+    """Check background_ids can be used.
+
+    This is a basic run through to check the behavior.
+    """
+
+    pha = DataPHA("src", [1, 2, 3], [1, 1, 1])
+    b1 = DataPHA("b1", [1, 2, 3], [1, 1, 1])
+    b2 = DataPHA("b2", [1, 2, 3], [1, 1, 1])
+
+    assert len(pha.background_ids) == 0
+
+    pha.set_background(b1)
+    assert pha.background_ids == pytest.approx([1])
```

Comment:
I always find it funny to use approz on integers, but I know you have your reasons.

Reply:
Can we just compare to `[1]` - I forget (in this case the return value should be a boring Python list and not a ndarray)?